op-service/sources/l1_beacon_client.go

```diff
@@ -108,18 +107,13 @@ func (cl *L1BeaconClient) GetBlobSidecars(ctx context.Context, ref eth.L1BlockRe
 		return nil, fmt.Errorf("%w: error in converting ref.Time to slot", err)
 	}
 
-	builder := strings.Builder{}
-	builder.WriteString(sidecarsMethodPrefix)
-	builder.WriteString(strconv.FormatUint(slot, 10))
-	builder.WriteRune('?')
-	v := url.Values{}
+	reqPath := sidecarsMethodPrefix + strconv.FormatUint(slot, 10)
```

Comment:
I don't love that `sidecarsMethodPrefix` has a trailing `/` while the other paths defined as constants do not - it creates a bit of a footgun when not building paths without path.Join`

Reply:
The others are `Method`, this one is `MethodPrefix`

Reply:
its a bit of a nit but there isn't really a reason to add two different types of constants, i would personally lean towards always using `path.Join` but not that big of a deal

Reply:
Changed it to use `path.Join`